src/styles/index.scss

```
@@ -1,3 +1,90 @@
+@use 'sass:math';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #fff;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__hand {
+    position: absolute;
+    left: 50%;
+    bottom: 50%;
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+  }
+
+  &__hand--minutes {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    bottom: 50%;
+    left: calc(50% - 1.5vmin);
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    animation: rotate-minutes 3600s steps(60) infinite;
+  }
+
+  &__hand--seconds {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    left: calc(50% - 0.75vmin);
+    transform-origin: bottom center;
+    transform: rotate(0deg);
+    animation: rotate-seconds 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+  }
+
+  &--speed-up {
+    .stopwatch__hand--minutes {
+      animation-duration: 600s;
+    }
+
+    .stopwatch__hand--seconds {
+      animation-duration: 10s;
+    }
+  }
+}
+
+@keyframes rotate-seconds {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotate-minutes {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
 }
```

Don't need to duplicate keyframes with the same scenario, use only one keyframes
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}
```